djangocms_repeater/templates/multiple-single-header.html

```diff
@@ -1,5 +1,5 @@
 {% load cms_tags %}
-<li class="masthead careers-page" style="background-image:url('{{instance.backgroundImage}}')">
+<li class="masthead careers-page" style="background-image:url('{% firstof instance.background_image.url instance.background_image_url %}')">
```

Comment:
This is the bit where it tries filerfield then if it is not available then try the urlfield.

I have renamed all existing UrlFields with "xxxx_url" and have FilerImageFieid calledl "xxxx" (I also change them to underscore instead of camel, which is slightly more python way and shows better on admin by default)